handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -158,20 +157,16 @@
     }
 
     private static final int MAX_PLAINTEXT_LENGTH = 16 * 1024; // 2^14
-    private static final int MAX_COMPRESSED_LENGTH = MAX_PLAINTEXT_LENGTH + 1024;
-    private static final int MAX_CIPHERTEXT_LENGTH = MAX_COMPRESSED_LENGTH + 1024;
+    private static final int MAX_CIPHERTEXT_LENGTH = MAX_PLAINTEXT_LENGTH + 1024;
 
-    // Header (5) + Data (2^14) + Compression (1024) + Encryption (1024) + MAC (20) + Padding (256)
+    // Header (5) + Data (2^14) + Encryption (1024) + MAC (20) + Padding (256)
+    // No compression supported!
```

Comment:
@Scottmitch btw do you think I should maybe factor out the disabling of compression explicitly in another PR just to make it more "clear" ?